saga/README.md

```diff
@@ -0,0 +1,121 @@
+# Saga Pattern
+
+This example demonstrates how to implement the [Saga pattern](https://microservices.io/patterns/data/saga.html) for realizing distributed transactions across multiple microservices, in a safe and reliable way using change data capture.
+
+Similarly to the outbox pattern, this implementation avoids unsafe dual writes to a service's database and Apache Kafka by channeling all outgoing messages through the originating service's database and capturing them from there using CDC and Debezium.
+
+There are three services involved:
+
+* _order-service:_ originator and orchestrator of the Saga
+* _customer-service:_ approves or rejects the customer's credit needed to fulfill an incoming order
+* _payment-service_ executes the payment associated to an incoming order
+
+## Running the Example
+
+Build and start up:
+
+```console
+$ mvn clean verify
+```
+
+```console
+$ docker-compose up --build
```

Comment:
since the compose file needs ENV var setting for DEBEZIUM_VERSION e.g. 1.4 it might be helpful to explicitly point this out here in the README somewhere?

Reply:
Done.